log-manager/src/main/java/io/airlift/log/LoggingMBean.java

```diff
@@ -15,58 +15,76 @@
  */
 package io.airlift.log;
 
-import ch.qos.logback.classic.Logger;
-import ch.qos.logback.classic.LoggerContext;
-import org.slf4j.LoggerFactory;
 import org.weakref.jmx.Managed;
 
+import java.util.Collections;
+import java.util.Locale;
 import java.util.Map;
-
-import static ch.qos.logback.classic.Level.toLevel;
-import static com.google.common.collect.Maps.newTreeMap;
-import static org.slf4j.Logger.ROOT_LOGGER_NAME;
+import java.util.TreeMap;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
 
 public class LoggingMBean
 {
+    private static final String ROOT_LOGGER_NAME = "";
+
     @Managed
+    @SuppressWarnings("MethodMayBeStatic")
     public String getLevel(String loggerName)
     {
-        return getLogger(loggerName).getEffectiveLevel().toString();
+        return getEffectiveLevel(getLogger(loggerName)).toString();
     }
 
     @Managed
+    @SuppressWarnings("MethodMayBeStatic")
     public void setLevel(String loggerName, String newLevel)
     {
-        getLogger(loggerName).setLevel(toLevel(newLevel));
+        getLogger(loggerName).setLevel(Level.valueOf(newLevel.toUpperCase(Locale.US)).toJulLevel());
```

Comment:
This class breaks the encapsulation of `Logging`. It seems like either everything here should go through `Logging`, or we simply say this class is part of the public interface. And it seems we already use the latter, so this is fine.